src/components/App.js

```diff
@@ -95,6 +101,12 @@ function App() {
                     autoComplete="off"
                 />
                 <div className="download-btn">
```

Comment:
The functionality works completely fine. 
Just to follow some standards, ```<div className="clearCode">``` should not be inside ```<div className="download-btn">``` as matter of fact, it is not a download button

Reply:
I agree but from the css point of view it is the correct place to be, otherwise I would need another container with the clearCode button and the download buttons.
How about just renaming `download-btn` to `btn-container`? What do you think?

Reply:
> How about just renaming download-btn to btn-container? What do you think?

Yes, that would be okay.